docs/modules/demos/pages/argo-cd-git-ops.adoc

```diff
@@ -0,0 +1,317 @@
+
+When managing all resources and configs via Git, deploying sensitive properties like certificates or credentials via Git becomes a problem.
+
+There are multiple solutions like Hashicorp or Bitwarden, which heavily depend on the infrastructure already available.
+
+For the sake of this demo, {sealed-secrets}[Bitnami's Sealed Secrets] are utilized. 
+Sensitive data is encrypted as a `SealedSecret` before commiting to the Git repository, synced via ArgoCD and decrypted by the Sealed Secrets controller into a standard Kubernetes `Secret`.
+
+This way, everything will be stored and managed in Git.
+
+== ArgoCD UI
+
+ArgoCD will be the first product that is deployed in this Demo. Once the pods are ready, you can port-forward the argocd-server in order to access the web UI.
+
+[source,console]
+----
+kubectl --namespace argo-cd port-forward service/argocd-server 8443:https
+----
+
+In your browser, go to `https://localhost:8443` and login with username `admin` and password `adminadmin`.
+
+NOTE: There will be an initial warning from the Browser, that this site is insecure due to self-signed certificates. This can be ignored in this case.
+
+The ArgoCD Web UI entry page shows an overview of deployed applications and their status and other metadata as the repository or the date of the last synchronization run.
+
+TODO: Screenshot - Argo UI overview
+
+Single applications can be inspected closer after clicking on e.g. the `airflow` project.
+
+TODO: Screenshot - Argo UI - Airflow
+
+Detailed information about the cluster, the cluster status and deployed components can be accessed in the application details.
+Additionally, if the Git repository and the cluster state itself differ, possible applied changes can be previewed in a code diff similar to Git pull requests.
+
+TODO: Screenshot - Argo UI - application details Network / List tab -> mark network tabs etc.
+
+Per default in this demo, the ArgoCD `Sync Policy` is set to `auto-sync`. This means that changes to the Git repository are immediatly synced into the cluster.
+This is nice in the demo case, but should be disabled for production use cases.
+
+TODO: Screenshot - Argo UI - applications/airflow/details
+
+Now, after a quick overview of the ArgoCD web UI, the following part demonstrates how to sync and deploy Stackable operators via ArgoCD.
+
+== Stackable operators
+
+The Stackable operators are deployed via ArgoCD using the Stackable Helm charts and an ArgoCD {argo-cd-applicationset}[`ApplicationSet`].
+`ApplicationSets` allow templating, which is required to e.g. manage and deploy to multi cluster environments (e.g. development - staging - production),
+using different versions and Git sources (repository & branch) as well as the possibility to deploy to different clusters.
+
+NOTE: This demo does not use a multi cluster environment for the sake of simplicity. 
+
+TODO: Do the code snippets actually help? Maybe text is better... its about showing at least some parts of a multi cluster setup...
```

Comment:
Maybe wrap the more technical details in a drop-down section, to signal that it is not crucial to understanding the demo, but also make that detail available for those who want to know?